### ISSUE TITLE
Data determining AVC Edge usage

### ISSUE THREAD

**Issue by umbernhard** [Sean-Case/WisconsinElection2016#1]
Hello!

I notice in your analysis that you cite places that use the AVC Edge for more than 25% of voters. How did you arrive at this number? I don't see it in the data set, outside of the WI SoS data. 

Cheers!

**Comment by Sean-Case** [comment]
Hi! 

I got the touchscreen use data from the GAB-190 March 2017 release on the Wisconsin Election Commission website. I'm on my phone atm so I don't have the URL to hand, but if you have trouble finding it, let me know.

Sean

**Comment by umbernhard** [comment]
No worries, I think I found it. I'd be curious to know how you stumbled
across it. I've spent a non-trivial amount of time in the WEC website and
didn't find it until I saw the URL in your R code.

Something to note about your analysis, the AVC Edge is serviced exclusively
by Command Central, who also operate in Minnesota. If the company were
compromised, I wonder if we'd see a similar effect in votes cast on their
machines in MN. Granted, MN doesn't use the Edge, which might make similar
favoring if trump in CC machines that much more damning.

-Matt

**Comment by umbernhard** [comment]
Another question, have you run this analysis for hand-counted ballots? They seem to follow a similar trend as the AVC Edge, and if I'm right that might suggest there's another confounding variable not being accounted for.

**Comment by Sean-Case** [comment]
Yes it would be interesting to see if there is a similar Command Central effect elsewhere!

So I checked your suggestion regarding paper counted ballots at municipality and reporting unit level. I made a variable of proportion of total ballots via paper and included it in the model. I found that although increasing paper proportion visually appears to have a small positive effect, this was not statistically significant. Also, the proportion of votes from AVC Edge use remained significant. Do you get the same result?

**Comment by umbernhard** [comment]
Cool. I unfortunately haven't gotten the chance to run my own analysis yet, but I will in a bit, and I'll definitely let you know what I come up with. 

One more question, what does the "hand" variable in ward.2016.2012.r.csv refer to? Hand-counted? Hand-recount?

**Comment by Sean-Case** [comment]
Hi,

If I remember correctly, these columns were created from the verified voting spreadsheet for presence of voting equipment in WI municipalities. I melted the equipment type column and then cast by something else I can't remember. Anyway yes, it indicates municipalities doing a hand count of ballots generally (not specifically for the recount)

I didn't use any of the verified voting derived columns in my final CSVs used for stat analysis or in any of the models.

Sean

P.s. for your previous question - I found the GAB files through an academic who I showed my early results to. I didn't come across it by searching or by chance :)
